CMakeLists.txt

```diff
@@ -317,7 +317,16 @@ else()
   add_subdirectory(3party/freetype)
   add_subdirectory(3party/icu)
 endif()
-add_subdirectory(3party/jansson)
+
+set(JANSSON_EXAMPLES OFF)
+set(JANSSON_BUILD_DOCS OFF)
+set(JANSSON_WITHOUT_TESTS OFF)
```

Comment:
Should it be ON to disable tests? We don't need them if we don't run them. Right?

Reply:
Fixed